### ISSUE TITLE
Can I sample K samples once?

### ISSUE THREAD

**Issue by guohe369** [LiyuanLucasLiu/ReinMax#1]
I want to get top-k mask instead of one-hot by logits and make the mask differential. Can I use reinmax? How?
